charts/kyverno/values.yaml

```diff
@@ -323,6 +326,20 @@ webhooksCleanup:
   # -- Node affinity constraints.
   nodeAffinity: {}
 
+  # -- Security context for the hook containers
+  securityContext:
+    runAsUser: 65534
+    runAsGroup: 65534
```

Comment:
@treydock do we need this in other security contexts ?

Reply:
I believe they have this, at least that's where I copied this from, was one of the other pod definitions with this defined

Reply:
👍